repo/packages/G/geoserver/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion" : "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name" : "geoserver",
+  "version" : "2.10.0-1.1.0",
```

Comment:
@ryadav88  Is there a specific version convention here? I've been using this package for a considerable amount of time in an offline production setting. My only reason to select 1.1.0 is that tracks with the versioning of the DCOS specific. Ref: https://github.com/appliedis/dcos-geoserver/releases/tag/1.1.0